src/controllers/v1/entities.js

```diff
@@ -6,36 +6,38 @@
  */
 
 // Dependencies
-const entitiesHelper = require(MODULES_BASE_PATH + "/entities/helper")
+const entitiesHelper = require(MODULES_BASE_PATH + '/entities/helper')
+const csv = require('csvtojson')
+const FileStream = require(PROJECT_ROOT_DIRECTORY + '/generics/file-stream')
+const entitiesQueries = require('../../databaseQueries/entities')
 
 /**
-   * entities
-   * @class
-*/
+ * entities
+ * @class
+ */
 
 module.exports = class Entities extends Abstract {
+	/**
+	 * @apiDefine errorBody
+	 * @apiError {String} status 4XX,5XX
+	 * @apiError {String} message Error
+	 */
 
-  /**
-   * @apiDefine errorBody
-   * @apiError {String} status 4XX,5XX
-   * @apiError {String} message Error
-   */
-
-  /**
-   * @apiDefine successBody
-   *  @apiSuccess {String} status 200
-   * @apiSuccess {String} result Data
-   */
+	/**
+	 * @apiDefine successBody
+	 *  @apiSuccess {String} status 200
+	 * @apiSuccess {String} result Data
+	 */
 
-  constructor() {
-    super("entities")
-  }
+	constructor() {
+		super('entities')
+	}
 
-  static get name() {
-    return "entities"
-  }
+	static get name() {
+		return 'entities'
+	}
 
-  /**
+	/**
 * @api {get} /entity/api/v1/entities/list List all entities
```

Comment:
@MallanagoudaB Why we need entities/list api signature description here? please remove this if not required